### DISCUSSION TITLE
Does it really support Marathi language

### DISCUSSION THREAD

**Discussion by unmeshvrije** [openai/whisper#1921] [Q&A]
I tried using whisper for a recording in Marathi language using the following command
`whisper audiocapture.mp3 --language Marathi --output_format srt --model medium`

I got some gibberish as the output and no errors.
Is there a special way to run whisper for languages such as Marathi ?
(I am running on MAC terminal)

**Comment by glangford** [comment]
The language is supported by whisper, but as you can see the error rate is high (due to lack of training data presumably)
https://github.com/openai/whisper#available-models-and-languages

* Try `--model large`
* See also this discussion
https://huggingface.co/spaces/openai/whisper/discussions/66
